plugins/modules/ecs_taskdefinition.py

```diff
@@ -321,7 +321,7 @@ def main():
         family=dict(required=False, type='str'),
         revision=dict(required=False, type='int'),
         force_create=dict(required=False, default=False, type='bool'),
-        containers=dict(required=False, type='list', elements='str'),
+        containers=dict(required=False, type='list', elements='list'),
```

Comment:
```suggestion
        containers=dict(required=False, type='list', elements='dict'),
```

I'm pretty sure it's a list if dicts rather than a list of lists: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.register_task_definition

Reply:
True. Let my try